treewide: remove bower (#443594)
NixOS/nixpkgs: doc/release-notes/rl-2511.section.md

- `mono4` and `mono5` have been removed. Use `mono6` or `mono` instead.

> - Everything related to `bower` got removed as the last in-tree user of it migrated away from it.

Reviewer: ```suggestion
+ Everything related to `bower` was removed, as it is deprecated and not used by anything in nixpkgs.
```
